change/@fluentui-react-tag-picker-9be006cd-1c03-46c8-a255-b35dd8971bee.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32317&commitId=47430bf84539c00b5fb17fd42de3e7dc0af6be27&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|16| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|6| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|

</details>


  <details><summary><b>TagPicker</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Disabled RTL.chromium.png|378| Changed|
|TagPicker.Disabled High Contrast.chromium.png|645| Changed|
|TagPicker.disabled.chromium.png|378| Changed|
|TagPicker.Disabled Dark Mode.chromium.png|393| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>